src/test/java/net/datafaker/base/InternetTest.java

```diff
@@ -146,6 +146,11 @@ void testPasswordMinLengthMaxLengthIncludeUpperCaseIncludeSpecial() {
         assertThat(faker.internet().password(10, 25, true, true)).matches("[a-zA-Z\\d!@#$%^&*]{10,25}");
     }
 
+    @Test
+    void testPort() {
+        assertThat(faker.internet().port()).isLessThanOrEqualTo(String.valueOf(65535));
```

Comment:
-1, -2, -3 and etc are also less than 65535 however should not be accepted here

Reply:
@snuyanzin What is a better way to compare?

Reply:
depends on use case if you want to compare numbers then you should compare numbers
if you want to compare strings then you should compare strings
right now you are comparing strings how it looks like you want to compare numbers

Reply:
Fixed

Reply:
Great! Btw, I'd recommend to use RepeatedTest instead of Test. Ideally with a parameter between 10 and 100, for example, RepeatedTest(10) or so. Anything above 100 will make the build a bit slower, but I find that for fast tests, 10-100 will find some issues quicker than just @Test. 

In any case, I think this code looks fine, even without the RepeatedTest

Reply:
@bodiam Replaced `@Test` to `@RepeatedTest(100)`

Reply:
the range in port generation and in test should be same otherwise there is still a chance to be flaky for the test